### PR TITLE
updated README to mention custom currencies and bouncing

### PR DESCRIPTION
**Pull request by khannanov-nil**
Closes https://github.com/NilFoundation/nil.js/issues/50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +146,48 @@ await wallet.syncSendMessage({
 
 It is only possible to perform sync calls within the confines of one shard.
 
+## Multi-currency and bouncing
+
+=nil; provides a multi-currency mechanism. A contract can be the owner of one custom currency, and owners can freely send custom currencies to other contracts. As a result, the balance of a given contract may contain standard =nil; tokens, and several custom tokens created by other contracts.
```
Comment (ukorvl) [inline README.md:151]: @KlonD90 wdyt maybe not mention `=nil; tokens` and just say `standard tokens`?

Comment (KlonD90) [reply README.md:151]: because it's not like standard token but our standard

Comment (ukorvl) [reply README.md:151]: I suggest to omit '=nil;', it looks like the name of the token, isn't it?

Comment (khannanov-nil) [reply README.md]: Got it! Removed =nil;

**Review by ukorvl — APPROVED** [review]

**Pull request merged**